composer.json

```diff
@@ -65,17 +18,20 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "yiisoft/arrays": "^3.0@dev",
-        "yiisoft/strings": "^3.0@dev",
+        "php": "^7.4",
         "ext-pdo": "*",
-        "php": "^7.4"
+        "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/strings": "^3.0@dev"
     },
     "require-dev": {
         "yiisoft/yii-console": "^3.0@dev",
         "yiisoft/active-record": "^3.0@dev",
```

Comment:
It's used in `GetTablesAliasTest` but that looks wrong since Active Record should not be involved in testing DB package.

Reply:
Done.